files/templates/alertmanager/notification-template.tmpl

```diff
@@ -43,11 +53,15 @@
 {{ if (index .Alerts 0).Annotations.opsrecipe -}}
 📗 Runbook: {{ template "__runbookurl" . }}
 {{ end -}}
-🔔 Alertmanager {{ template "__alertmanagerurl" . }}
 {{ if (index .Alerts 0).Annotations.dashboard -}}
 📈 Dashboard: {{ template "__dashboardurl" . }}
 {{ end -}}
+[[- if .MimirEnabled ]]
+👀 Details: {{ template "__queryurl" . }}
```

Comment:
```suggestion
👀 Query: {{ template "__queryurl" . }}
```

Reply:
I proposed `Details` in case of Mimir because the Grafana link is provided information on the query but also on the alert.

Reply:
Maybe we should rename that Explore then?

Reply:
You prefer query ?